### PR TITLE
feat: add option to polyfillsConfig to disable hashes in filenames

### PR DESCRIPTION
**Pull request by funglaub**
Solves #800

### CODE REVIEW

File: packages/building-utils/index-html/create-index-html.js [hunk 1]
```diff
@@ -57,6 +58,7 @@ const { cleanImportPath } = require('./utils');
  * @property {boolean} [systemJs] whether to polyfill systemjs
  * @property {boolean} [systemJsExtended] whether to polyfill systemjs, extended version with import maps
  * @property {boolean} [esModuleShims] whether to polyfill es modules using es module shims
+ * @property {boolean} [skipHash] whether to skip the content hash for the emitted file
```
Comment (LarsDenBakker) [inline packages/building-utils/index-html/create-index-html.js]: I think it's clearer to call this `hashPolyfills`, with a default of true.

**Comment by LarsDenBakker** [comment]
I'm not sure why the CI is failing now

**Comment by funglaub** [comment]
@LarsDenBakker it's fixed now. Was a bit tricky to find the error.

File: packages/karma-esm/src/test-loader-browser-script.js [hunk 2]
```diff
@@ -30,9 +30,20 @@ const {
  * @param {import('@open-wc/building-utils/index-html/create-index-html').Polyfill[]} polyfills
  * @param {object | null} [importMap]
  */
+
+const defaultPolyfillsConfig = {
```
Comment (LarsDenBakker) [inline packages/karma-esm/src/test-loader-browser-script.js]: was this already the default value?

**Pull request merged**